### ISSUE TITLE
websocket error

### ISSUE THREAD

**Issue by Dikshuy** [huawei-noah/SMARTS#890]
** High-Level Description**
after the simulations are completed in localhost, terminal give WebSocket error

**Desired SMARTS version**
0.4.12

**Operating System**
ubuntu 18.04

**Problems**
Once the episodes are completed, it's showing this error:
```
 ERROR:websocket:error from callback <function Client._connect.<locals>.on_close at 0x7f4c33a72e60>: on_close() takes 1 positional argument but 3 were given
```
I have to press ctrl+c to stop the execution.

**Comment by Gamenot** [comment]
Hello @Dikshuy , sorry for the late reply. Since 0.4.12 is still a patch version we tend not to go backwards to fix problems, however, I could understand why this would be an irritation if you are trying to automate a number of tests. Do you find this is still a problem in SMARTS 0.4.16?

**Comment by Dikshuy** [comment]
I am using the latest version ig. It arouses when I try to use localhost. It takes some time to load on the server. That's why it is causing some trouble.

**Comment by Gamenot** [comment]
OK, I have confirmed the problem with this method. One of the libraries we are using has a misleading description of what it provides back to us. However, I suspect that this would not be enough to prevent the program from exiting. Did you notice anything else along with this problem that might indicate that something is wrong?

**Comment by Dikshuy** [comment]
I have to press `ctrl+c` every time I need to kill the process after this error occurs. That's it.

**Comment by Adaickalavan** [comment]
Issue closed through pull request #906 .
